app/components/provider_interface/application_header_components/offer_will_be_declined_by_default_component.html.erb

```diff
@@ -2,9 +2,5 @@
   Waiting for candidate’s response
 </h2>
 <p class="govuk-body">
-  <% if application_choice.continuous_applications? %>
-    <%= continuous_applications_offer_text %>
-  <% else %>
```

Comment:
We still have some candidates with offers for 2023 – they have till ~12th oct until DBD

Reply:
Done in https://github.com/DFE-Digital/apply-for-teacher-training/pull/8644/commits/f33140e68ab3580ac2df317d4ece637e48893e13. Added it as a separate commit as we can re-revert it later.